src/components/stepforms/widgets/FilterSimpleCondition.vue

```diff
@@ -87,13 +87,19 @@ export default class FilterSimpleConditionWidget extends Vue {
   })
   value!: FilterSimpleCondition;
 
+  @Prop({
+    type: Array,
+    default: () => [],
+  })
+  columnNames!: string[];
+
   @Prop({ type: String, default: '' })
   dataPath!: string;
 
   @Prop({ type: Array, default: () => [] })
   errors!: ErrorObject[];
 
-  @VQBModule.Getter columnNames!: string[];
+  @VQBModule.Getter('columnNames') columnNamesGetter!: string[];
```

Comment:
Hm...This is gonna cause an error in Tucana in the console (like `getter columnNames` does not exist).
Let's keep that way for now and ignore that.

Could you rather name it `columnNamesFromStore`? Then you can use a computed property that will use the prop is present and fallback to the store one.
In order to do that, avoid using any default value for the prop.